src/main/scala/interfaces/Verification.scala

```diff
@@ -7,13 +7,52 @@
 package viper.silicon.interfaces
 
 import viper.silicon.interfaces.state.Chunk
+import viper.silicon.state.terms.Term
 import viper.silicon.state.{State, Store}
 import viper.silver.verifier.{Counterexample, Model, VerificationError}
-import viper.silicon.state.terms.Term
 
 /*
  * Results
  */
+class VerificationResultWrapper(verificationResultsarg: Seq[VerificationResult]) {
```

Comment:
Please replace `verificationResultsarg` and accompanying `def verificationResults` by just parameter `val verificationResults`